plenum/test/test_node.py

```diff
@@ -145,7 +145,7 @@ def __init__(self, *args, **kwargs):
             notifierEventTriggeringConfig=notifierEventTriggeringConfig,
             pluginPaths=pluginPaths)
         for i in range(len(self.replicas)):
```

Comment:
It would be more clear to iterate over the keys.

Reply:
Done